src/main/java/org/broadinstitute/consent/http/models/DarCollection.java

```diff
@@ -0,0 +1,77 @@
+package org.broadinstitute.consent.http.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.sql.Timestamp;
+
+//represents a multi-dataset access request
+public class DarCollection {
+
+  @JsonProperty
+  Integer darCollectionId;
+
+  @JsonProperty
+  String darCode;
+
+  @JsonProperty
+  Timestamp createDate;
```

Comment:
These should be dates for consistency

Reply:
Okay I can make that change! but I'm wondering what you mean, consistency with what? Because the data access requests use timestamps

Reply:
You are correct, we are actually all over the place. `Dac`, `Institution`, `Vote`, `Dataset`, `User`, `Match`, `Election`, and others all use `Date`, but `Consent` and `DataAccessRequest` use `Timestamp`. 

I'm not sure if we ever had an intentional approach to the actual class, but reading up on it, both are flawed and [`Instant`](https://docs.oracle.com/javase/10/docs/api/java/time/package-summary.html) is [now the preferred time storage class](https://www.baeldung.com/migrating-to-java-8-date-time-api). Let's stick with `Timestamp` for now and we can file a ticket to explore looking at migrating all of these fields to `Instant` if that makes more sense.